tools/commands/run.cpp

```diff
@@ -11,24 +11,62 @@ namespace evmc
 {
 namespace cmd
 {
+namespace
+{
+constexpr auto deploy_address = 0x000000000000000000000000000000000c9ea7ed_address;
```

Comment:
Why does this address has so many leading zeroes?

Reply:
Changed.